helper/validate.js

```diff
@@ -0,0 +1,59 @@
+/** validate userInput */
+export default class ValidateUser {
+/**
+ *
+ * @param {*} str
+ * @returns {*} return
+ * validate username field input *
+ */
+  static validateEmail(str) {
+    if (!str || str.trim().length < 1) {
+      return { error: 'Email cannot be empty' };
+    }
+
+    const emailRegex = /^\w+([.-]?\w+)@\w+([.-]?\w+)(\.\w{2,3})+$/;
+    if (!emailRegex.test(str)) {
+      return { error: 'Invalid email format' };
+    }
+    return {};
+  }
+
+  /**
+ *
+ * @param {*} str
+ * @returns {*} return
```

Comment:
you should specify what is returned inside the curly brace, in this case, object. then outside it, you can give a one line statement explaining what is inside.